frontend/prisma/migrations/20221220153624_add_results/migration.sql

```diff
@@ -0,0 +1,37 @@
+/*
+  Warnings:
+
+  - You are about to drop the `Post` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `Profile` table. If the table is not empty, all the data it contains will be lost.
+  - You are about to drop the `User` table. If the table is not empty, all the data it contains will be lost.
+
+*/
+-- CreateEnum
+CREATE TYPE "ResultSource" AS ENUM ('WEB', 'EMBED');
+
+-- DropForeignKey
+ALTER TABLE "Post" DROP CONSTRAINT "Post_authorId_fkey";
+
+-- DropForeignKey
+ALTER TABLE "Profile" DROP CONSTRAINT "Profile_userId_fkey";
+
+-- DropTable
+DROP TABLE "Post";
+
+-- DropTable
+DROP TABLE "Profile";
+
+-- DropTable
+DROP TABLE "User";
+
+-- CreateTable
+CREATE TABLE "Result" (
+    "id" STRING NOT NULL,
+    "updateToken" STRING NOT NULL,
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "source" "ResultSource" NOT NULL DEFAULT 'WEB',
+    "embedSourceUrl" STRING,
+    "value" JSONB NOT NULL,
```

Comment:
Ještě by tady mělo být ID té kalkulačky, ke které to ukládám.

Reply:
calculatorID as string

Reply:
Nevidím to tam?

Reply:
už